src/invidious/helpers/youtube_api.cr

```diff
@@ -30,9 +30,49 @@ def request_youtube_api_browse(continuation)
     "continuation": continuation,
   }
 
+  return _youtube_api_post_json("/youtubei/v1/browse", data)
+end
+
+####################################################################
+# request_youtube_api_search(search_query, params, region)
+#
+# Requests the youtubei/vi/search endpoint with the required headers
```

Comment:
```suggestion
# Requests the youtubei/v1/search endpoint with the required headers
```

Reply:
Oh, right, thanks :) I'll fix those later in a separate commit!